mkosi/distributions/gentoo.py

```diff
@@ -99,7 +99,7 @@ def install(cls, state: MkosiState) -> None:
 
         with urllib.request.urlopen(stage3tsf_path_url) as r:
             # e.g.: 20230108T161708Z/stage3-amd64-nomultilib-systemd-mergedusr-20230108T161708Z.tar.xz
-            regexp = rf"^[0-9]+T[0-9]+Z/stage3-{arch}-nomultilib-systemd-mergedusr-[0-9]+T[0-9]+Z\.tar\.xz"
+            regexp = rf"^[0-9]+T[0-9]+Z/stage3-{build_arch}-llvm-systemd-mergedusr-[0-9]+T[0-9]+Z\.tar\.xz"
```

Comment:
`build_arch` doesn't seem to be defined.

Reply:
fixed